src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -496,8 +498,12 @@ class ConnectionsList extends React.Component<ConnectionsListProps, { scores: Di
     }
 
     private compareLinks = (a: FatLinkType, b: FatLinkType) => {
-        const aText = chooseLocalizedText(a.label, this.props.lang).text.toLowerCase();
-        const bText = chooseLocalizedText(b.label, this.props.lang).text.toLowerCase();
+        const aText = (a.label.length > 0 ?
```

Comment:
I think you should extract this to a free function (because there're at least 10 places with exactly the same block).